### DISCUSSION TITLE
New maintainers wanted!

### DISCUSSION THREAD

**Discussion by ArtyIF** [GradienceTeam/Gradience#71] [Announcements]
I gotta be honest, I lost interest in this project. I want other people to help me maintain it. I transferred this repository to an organization, and if anyone wants to become a member of the organization, comment here.

**Comment by ArtyIF** [comment]
Also, I have a branch of the program with unfinished plugin support, if anyone wants to maintain that and then merge it into the main repository I'll be glad.

**Comment by 0xMRTT** [comment]
I really like this project. It's very useful for customisation. I can help you.

**Reply by tseli0s** [reply]
I would be happy to contribute to other projects, especially this which I use for some time now. I could be some help

**Comment by daudix** [comment]
This is a bit sad, but I understand you, I want to keep this project alive and continue to contribute.
